{"classifiers": ["Development Status :: 5 - Production/Stable", "Environment :: X11 Applications :: GTK", "Framework :: Tryton", "Intended Audience :: End Users/Desktop", "License :: OSI Approved :: GNU General Public License (GPL)", "Natural Language :: Bulgarian", "Natural Language :: Catalan", "Natural Language :: Chinese (Simplified)", "Natural Language :: Czech", "Natural Language :: Dutch", "Natural Language :: English", "Natural Language :: Finnish", "Natural Language :: French", "Natural Language :: German", "Natural Language :: Hungarian", "Natural Language :: Indonesian", "Natural Language :: Italian", "Natural Language :: Persian", "Natural Language :: Polish", "Natural Language :: Portuguese (Brazilian)", "Natural Language :: Russian", "Natural Language :: Slovenian", "Natural Language :: Spanish", "Natural Language :: Turkish", "Natural Language :: Japanese", "Operating System :: OS Independent", "Programming Language :: Python :: 3", "Programming Language :: Python :: 3.5", "Programming Language :: Python :: 3.6", "Programming Language :: Python :: 3.7", "Topic :: Office/Business"], "download_url": "http://downloads.tryton.org/5.4/", "extensions": {"python.details": {"contacts": [{"email": "bugs@tryton.org", "name": "Tryton", "role": "author"}], "document_names": {"description": "DESCRIPTION.rst"}, "project_urls": {"Home": "http://www.tryton.org/"}}}, "extras": ["calendar"], "generator": "bdist_wheel (0.29.0)", "keywords": ["business", "application", "ERP"], "license": "GPL-3", "metadata_version": "2.0", "name": "tryton", "platform": "any", "project_url": "Bug Tracker, https://bugs.tryton.org/", "requires_python": ">=3.5", "run_requires": [{"extra": "calendar", "requires": ["GooCalendar (>=0.5)"]}, {"requires": ["PyGObject", "pycairo", "python-dateutil"]}], "summary": "Tryton desktop client", "version": "5.4.1"}